### ISSUE TITLE
[0781 PS] Ensure all old pages / logic will be hidden by new flipper

### ISSUE THREAD

**Issue by SamStuckey** [department-of-veterans-affairs/va.gov-team#98413]
## Context
In addition to adding a new 0781 form flow, we are moving the entire form flow relative to the rest of the 526.

## Acceptance Criteria
This applies to both
- old pages that are part of the 0781 flow will be hidden when `syncModern0781Flow === true`
- conditional logic that depends on pages that have been _moved_ will be updated accordingly
  - part of the work here is going through the old flow and new flow and ensuring we don't have any unintented overlap

**Comment by freeheeling** [comment]
Confirm that [[3.1e] Start page - empty state](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9805-33618&t=4zN8OfZ9820SyPru-4) should be displayed in any of the following conditions:
- user didn't select/claim a mental health condition on [[2.1] 0781 screener](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9250-79794&t=4zN8OfZ9820SyPru-4)
- `syncModern0781Flow === false`
- claim is not new
- claim is not for a new disability

In other words, the Additional forms section should render regardless of whether or not `syncModern0781Flow === true`

**Comment by freeheeling** [comment]
Clarify what is meant by _moved_ in this item from the ticket description's AC:
> conditional logic that depends on pages that have been _moved_ will be updated accordingly

**Comment by SamStuckey** [comment]
@freeheeling TL;DR on this ticket is just "add the flipper to the old pages"

**Comment by SamStuckey** [comment]
With a note to consider the 'Additional Forms' (aka start-page) flipper. Right now it's just flipping based on the 0781 flipper, with a comment to probably update that

**Comment by SamStuckey** [comment]
> moved

the 0781 flow now exists in the 'additional forms' chapter. previously there was no additional forms chapter so the 0781 has been "moved" into it.

**Comment by lisacapaccioli** [comment]
@NB28VT Do we need this ticket anymore? I feel like the work you had done already was the same thing.
